packages/builtin-components/demo/demo/radar.demo.js

```diff
@@ -6,15 +6,16 @@ const {
 } = builtinComponents;
 
 const source = [
-  ['product', '2015', '2016', '2017'],
-  ['Matcha Latte', 43.3, 85.8, 93.7],
-  ['Milk Tea', 83.1, 73.4, 55.1],
-  ['Cheese Cocoa', 86.4, 65.2, 82.5],
-  ['Walnut Brownie', 72.4, 53.9, 39.1],
+  {product: 'Matcha Latte', 2015: 43.3, 2016: 85.8, 2017: 93.7},
+  {product: 'Milk Tea', 2015: 83.1, 2016: 73.4, 2017: 55.1},
+  {product: 'Cheese Cocoa', 2015: 86.4, 2016: 65.2, 2017: 82.5},
+  {product: 'Walnut Brownie', 2015: 72.4, 2016: 53.9, 2017: 39.1},
 ];
 
+
+
 export default function RadarDemo() {
   return (
-    <Radar source={source} />
+    <Radar source={source}/>
```

Comment:
Please read this document and integrate eslint to your VSCode which help to check your coding styles.

https://github.com/Microsoft/vscode-eslint